plugins/powerd/fu-plugin-powerd.c

```diff
@@ -61,6 +79,19 @@ fu_plugin_startup (FuPlugin *plugin, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_plugin_inhibit_suspend(GError **error)
```

Comment:
Can we make this fu_plugin_powerd_create_suspend_file() and move it above the other function for symmetry